cordova/config/config.template

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="org.montelibero.solar" version="$PACKAGE_VERSION" android-versionCode="$ANDROID_VERSIONCODE" ios-CFBundleVersion="$IOS_BUNDLE_VERSION" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
-    <name>Solar</name>
+    <name>MTL Solar</name>
     <description>
-        Wallet for the Stellar payment network by SatoshiPay.
+        Wallet for the Stellar ecosystem by Montelibero.
     </description>
     <author email="hello@solarwallet.io" href="https://solarwallet.io/">
         SatoshiPay Ltd
```

Comment:
probably need to change email, href and author name as well? Not sure where these would be used

Reply:
I'll roll back the changes here